tests/readme.rs

```diff
@@ -0,0 +1,68 @@
+mod fixtures;
+
+use fixtures::{server, Error, TestServer, DIRECTORIES, FILES};
+use rstest::rstest;
+use select::predicate::Attr;
+use select::{document::Document, node::Node};
+
+#[rstest]
+/// Do not show readme contents by default
+fn no_readme_contents(server: TestServer) -> Result<(), Error> {
+    let body = reqwest::blocking::get(server.url())?.error_for_status()?;
+    let parsed = Document::from_read(body)?;
+    for &file in FILES {
+        assert!(parsed.find(|x: &Node| x.text() == file).next().is_some());
+    }
+    for &dir in DIRECTORIES {
+        assert!(parsed.find(|x: &Node| x.text() == dir).next().is_some());
+    }
+    assert!(parsed.find(Attr("id", "readme")).next().is_none());
+    assert!(parsed.find(Attr("id", "readme-filename")).next().is_none());
+    assert!(parsed.find(Attr("id", "readme-contents")).next().is_none());
+
+    Ok(())
+}
+
+#[rstest(
+    dir,
+    content,
+    case("", "Test Hello Yes"),
+    case("/dira/", "This is dira/readme.md"),
+    case("/dirb/", "This is dirb/readme.md")
```

Comment:
```suggestion
    case("/dirb/", "This is dirb/README.md")
```
Let's test uppercase as well.

Reply:
Any idea how to test both?

for now I think to test uppercase would be problematic, having both "README.md" and "readme.md" (thought might not exactly be undefined behavior) there is no priority for one over another. And fixtures is automatically making the file "readme.md" on all directories, so adding "README.md" in the `FILES` might make things confusing to test.

Reply:
You could put the uppercased one in a different directory.